core/metacling/src/TClingCallFunc.cxx

```diff
@@ -528,6 +623,433 @@ void TClingCallFunc::make_narg_ctor_with_return(const unsigned N, const string &
    buf << "}\n";
 }
 
+int TClingCallFunc::generate_wrapper(std::string& wrapper_name, std::string& wrapper) {
+  const FunctionDecl *FD = GetDecl();
```

Comment:
We use fInterp, fNormCtxt and the Decl, so we actually use every single data member in this method :)